cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/batch/sinks/TimePartitionedFileSetDatasetAvroSink.java

```diff
@@ -40,43 +43,64 @@
 import org.apache.hadoop.mapreduce.Job;
 
 import java.util.Map;
+import javax.annotation.Nullable;
 
 /**
  * A {@link BatchSink} to write Avro record to {@link TimePartitionedFileSet}
  */
+@Plugin(type = "sink")
+@Name("TimePartitionedFileSetDatasetAvroSink")
```

Comment:
That's a pretty long name :) Can it be `TPFSAvroSink` and use the description to tell more details?